charts/tractusx-connector/values.yaml

```
@@ -253,12 +253,12 @@ controlplane:
   # choice for the user. This also increases chances charts run on environments with little
   # resources, such as Minikube. If you do want to specify resources, uncomment the following
   # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-  # limits:
-  #   cpu: 100m
-  #   memory: 128Mi
-  # requests:
-  #   cpu: 100m
-  #   memory: 128Mi
+  limits:
+    cpu: 1.2
```

`1.2` looks strange to me :D I've never seen a helm chart where someone set something like that. I would guess that most are using 0.5 or even 1 increments. Also 1 vs 1.2 won't make that much difference.

```suggestion
    cpu: 1
```

---

Changed upwards to 1.5 instead to avoid small spikes in processing power casing issues.